crates/iota-rest-kv/src/routes/health.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use axum::{Json, extract::State, response::IntoResponse};
+use serde::Serialize;
+
+use crate::{services::AwsStatus, types::SharedKvStoreService};
+
+bin_version::bin_version!();
+
+/// Represent a health status response of the REST API server
+#[derive(Serialize)]
+pub struct HealthResponse {
+    /// The REST API status
+    pub status: String,
+    /// Version of the binary
+    pub version: String,
+    /// The Git hash of the binary
+    pub git_hash: String,
+    /// The total uptime of the REST API server
+    pub uptime: String,
+    /// The status of AWS compoenents the REST API rely to properly function
```

Comment:
```suggestion
    /// The status of AWS components the REST API rely to properly function
```
We need to enable the CI (e.g. searching for typos) otherwise such errors will cause a failure after merging.